python/cuml/cuml/tests/test_umap.py

```diff
@@ -838,3 +838,29 @@ def test_umap_distance_metrics_fit_transform_trust_on_sparse_input(
 
     if umap_learn_supported:
         assert array_equal(umap_trust, cuml_trust, 0.05, with_sign=True)
+
+
+@pytest.mark.parametrize("n_rows", [3000])
+@pytest.mark.parametrize("n_features", [200, 500])
```

Comment:
any reason to test with these two specific sizes?

Reply:
tbh no specific reason..!